example_buffer.py

```diff
@@ -177,6 +177,9 @@ def _determine_chunk_to_make(write_dir):
 
     return chunk_to_make, False
 
+def get_window_size(chunk_num):
+    return min(500000, (chunk_num + 5) * MINIMUM_NEW_GAMES)
```

Comment:
I think this should scale sublinearly with chunk_number, since otherwise it's possible for the window size to just stay exactly at first game ~ latest game during the early phases.

So something like `min(500_000, (chunk_num + 5) * MINIMUM_NEW_GAMES * 0.5)`?

Reply:
ah but here's a trick, it already is sublinear :)  Games still get finished during the time it takes to write out the chunks, which is ~minutes.  we end up with about 1.5x-2x MINIMUM_NEW_GAMES per model.

Reply:
Yeah, I understand that that's what's happening - but it'd be nice to have that implicit assumption be in the code, so that we don't have weird cascading effects due to "training happens to have gotten faster and now our pipeline isn't working for some reason"

Reply:
Done.